src/componentes/CardList/CardList.js

```diff
@@ -1,14 +1,24 @@
 import React from "react";
 import ratings from '../../ratings.json';
 import Card from "../card/card";
+import '../CardList/cardList.css';
 
 function CardList(props) {
-
-  const cardMap = ratings.map( 
+  const filterRatings = ratings.filter( 
+    (local) => local.ratings[0].category.name === props.category)
+  const cardMap = filterRatings.map( 
     (local) => <Card local={local}></Card>
```

Comment:
podemos separar un poco estas dos?
```suggestion
  const filterRatings = ratings.filter( 
    (local) => local.ratings[0].category.name === props.category)
    
  const cardMap = filterRatings.map( 
    (local) => <Card local={local}></Card>
```